CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
 - Speakerdeck embedding
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
+- We are now using highlight.js instead of highlight.js+prism.js. The highlighting for the following languages has been dropped:
+    - tiddlywiki
+    - mediawiki
+    - jsx
```

Comment:
I would add a space between highlight.js+prism.js (see suggestion above).
And maybe add how many new languages we support instead (if any).

Reply:
better?